packages/sources/amberdata/src/config.ts

```diff
@@ -21,3 +24,36 @@ export const makeConfig = (prefix = ''): Config => {
   config.api.baseURL = config.api.baseURL || DEFAULT_API_ENDPOINT
   return config
 }
+
+export const makeWSHandler = (): WSSubscriptionHandler => {
+  const config = makeConfig()
+  return {
+    connection: {
+      url: config.api.baseWsURL || DEFAULT_WS_API_ENDPOINT,
+      protocol: { headers: { ...config.api.headers }}
+    },
+    subscribe: (input) => {
+      // TODO: What to do with different enpoints? Should every endpoint have its own WS Handler?
```

Comment:
I believe that's what the multiplexing behaviour does for us for free, every endpoint gets its own subscription, which is multiplexed onto the single connection we have to the api provider

Reply:
This was more about the subscription message necessary for each endpoint. Probably `price` and `marketCap` would have different subscription messages